packages/fxa-content-server/app/scripts/lib/metrics.js

```diff
@@ -53,8 +54,8 @@ const ALLOWED_FIELDS = [
   'migration',
   'navigationTiming',
   'numStoredAccounts',
-  'plan_id',
-  'product_id',
+  'planId',
+  'productId',
```

Comment:
Most of the properties in here a camel-case, so I renamed these for consistency.